### DISCUSSION TITLE
jax.interpreters.xla._DeviceArray back to numpy-array

### DISCUSSION THREAD

**Discussion by Anselmoo** [jax-ml/jax#10547] [Q&A] [answered]
Dear `jax`-developer and community,

thx for accelerating array operations.

I am wondering if there is an option to switch back the type from `DeviceArray` back to `Numpy`, like

```python

jnp.asarray().to_numpy()
```

or so.

That would be helpful to avoid `raise` crashes in KERAS:

```shell
TypeError: Expect the data to ImageInput to be numpy.ndarray or tf.data.Dataset, 
but got <class 'jaxlib.xla_extension.DeviceArray'>.
```

**Comment by mattjj** [comment, marked as answer]
Thanks for the question!

Just call `np.array(jax_array)` where `np` is from `import numpy as np`.

Does that answer your question?

**Reply by Anselmoo** [reply]
Tested and checked @mattjj 

thx, it works!
